test/distributed/_composable/fsdp/test_fully_shard_compile.py

```diff
@@ -243,7 +243,7 @@ def _check_count(copy_count, resize_count):
                 f"Unexpected number of `inductor.resize_storage_bytes_` ops (expected {resize_count}, got {actual_resize_count}) in graph: {graph}",  # noqa: B950
             )
 
-        if not torch._dynamo.compiled_autograd.in_compiled_autograd_region:
+        if not torch._dynamo.compiled_autograd.local.get("in_compiled_autograd_region"):
```

Comment:
Wonder should we encapsulate `in_compiled_autograd_region` into a function like `torch._dynamo.compiled_autograd.in_compiled_autograd_region()` to avoid needing to type `.local.` in user code? (similarly for `compiled_autograd_enabled`)?

(in my experiment enabling Traceable FSDP for Ads model I ended up needing check both flags, and simplifying them would be nice)